packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -54,19 +58,22 @@ const LinkControlSearchInput = ( {
 	}
 
 	return (
-		<form onSubmit={ selectSuggestionOrCurrentInputValue }>
+		// eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+		<form
+			onSubmit={ selectSuggestionOrCurrentInputValue }
+			onKeyDown={ ( event ) => {
+				if ( event.keyCode === ENTER ) {
+					return;
+				}
+				handleLinkControlOnKeyDown( event );
+			} }
+			onKeyPress={ handleLinkControlOnKeyPress }
+		>
 			<URLInput
 				className="block-editor-link-control__search-input"
 				value={ value }
 				onChange={ selectItemHandler }
-				onFocus={ selectItemHandler }
-				onKeyDown={ ( event ) => {
-					if ( event.keyCode === ENTER ) {
-						return;
-					}
-					handleLinkControlOnKeyDown( event );
-				} }
-				onKeyPress={ handleLinkControlOnKeyPress }
+				onFocus={ noop }
```

Comment:
Does this need to be assigned at all?

Reply:
I was getting some console error because of it not being assigned. I couldnt repro so I took this out.